angular/projects/admin/src/app/services/theme-template.service.ts

```diff
@@ -23,38 +23,61 @@ export class ThemeTemplateService {
       .valueChanges();
   }
 
-  getTemplatesForTheme(theme: string) {
+  getReference(themeId: string, templateId: string) {
```

Comment:
I think this is better to move it under getTemplatesForTheme for consitency with another service that using infinite scroll